h2o-core/src/main/java/water/H2O.java

```diff
@@ -573,6 +595,18 @@ private static void validateArguments() {
       if (H2O.ARGS.login_conf == null) {
         parseFailed("Must specify -login_conf argument");
       }
+    } else {
+      if (H2O.ARGS.form_auth) {
+        parseFailed("No login method specified. Specify one of hash/ldap/kerberos/pam login methods with -form_auth");
```

Comment:
Maybe I should rephrase the message. What I am trying to say is that when you enable form based authentication you also need to enable one of the available login methods. Otherwise H2O will not require login and the auth_form option doesn't apply.